Cmake 3.22 no longer accepts the --build option (in favour of -B)

Good afternoon! On Linux Mint 21.1 (based on Ubuntu 22.04 LTS), the current version of CMake is 3.22, which changes the building instructions slightly:

```
mkdir build
cmake -G "Unix Makefiles" -B build/
cd build/
make -j<YOUR # OF CORES>
cpack -G "DEB"
sudo dpkg -i scantailor-advanced-1.0.18-Linux.deb
```

The old instruction produces this error:
```
cmake -G "Unix Makefiles" --build ..
CMake Error: Unknown argument --build
CMake Error: Run 'cmake --help' for all supported options.
```

It would be wise to create a separate alternative instruction for those systems where `cmake` does not support the `build` argument.

--- a/README.md
+++ b/README.md
@@ -251,9 +251,10 @@ or [build them from sources](#building-from-sources).
 1. Navigate to the `scantailor-advanced-x.x.x` directory (`cd "<dir>"`) and enter:
 
    ~~~~
-   mkdir build; cd build
-   cmake -G "Unix Makefiles" --build ..
-   make -j `nproc`
+   mkdir build
+   cd build
+   cmake ..
+   cmake --build .
    ~~~~
 
 2. Optionally, install ScanTailor Advanced with:

